test/feature-benchmark/scenarios.py

```diff
@@ -881,3 +881,51 @@ def benchmark(self) -> MeasurementSource:
 """
             + str(self.n())
         )
+
+
+class PgCdc(Scenario):
+    pass
+
+
+class PostgresInitialLoad(PgCdc):
+    """Measure the time it takes to read 1M existing records from Postgres
+    when creating a materialized source"""
+
+    def shared(self) -> Action:
+        rows = ",".join(f"({i}, \'{i}')" for i in range(0, self.n()))
+        return TdAction(
+            f"""
+$ postgres-execute connection=postgres://postgres:postgres@postgres
+ALTER USER postgres WITH replication;
+DROP SCHEMA IF EXISTS public CASCADE;
+CREATE SCHEMA public;
+
+DROP PUBLICATION IF EXISTS mz_source;
+CREATE PUBLICATION mz_source FOR ALL TABLES;
+
+CREATE TABLE pk_table (pk INTEGER PRIMARY KEY, f2 TEXT);
+INSERT INTO pk_table VALUES {rows}
```

Comment:
one problem with cramming the row ingestion through this action is that if we ever print the output... it's pretty overwhelming. haven't dug into it, but I'm not sure why sometimes it's printed and sometimes it's not. 

might also be worth considering adding a `pgbench-execute` or `postgres-create-rows` action to populate the table

Reply:
hm, maybe you can construct a query here in the form of `INSERT INTO pk_table SELECT foo FROM generaties_series(..)` so that it's always compact and the rows are generated in pg

Reply:
Oh that's perfect, let me try that